scripts/convert.py

```diff
@@ -40,6 +45,37 @@ def __init__(self, arguments):
         self.post_process = PostProcess(arguments)
         self.verify_output = False
 
+        ##initialize last_used face to whichever face is first
+        self.matched_face = 0
+        ##load images for target face id
+        try:
+            models_dir = self.args.model_dir
+            target_images = [face_recognition.load_image_file(file) for file in glob.glob(models_dir + '**\\*.png', recursive=True)]
+        except:
+            print("no images found")
+            quit()
+
+        ##create target face encodings which will match closest
+        try:
+            self.target_encodings = [face_recognition.face_encodings(image)[0] for image in target_images]
+        except:
+            print("I wasn't able to locate any faces in at least one of the images. Check the image files. Aborting...")
+            quit()
+
+        ##load trained models, should have one model per image
+        try:
```

Comment:
one model per image ?! a 3.5 minute video clip will have 5,000 models ?!

Reply:
no no no sorry, one model per face encoding image. each model is trained the same way as before with however many pictures you like of source and target - this just associates each model with a face so we know which model to use once we identify a face.